CommonTools/PileupAlgos/interface/PuppiAlgo.h

```diff
@@ -2,6 +2,7 @@
 #define CommonTools_PileupAlgos_PuppiAlgo_h
 
 #include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
```

Comment:
well, if I dont include it, `scram b` does not compile. I get errors like:
```
 error: 'edm::ConfigurationDescriptions' has not been declared
```

Reply:
Well, maybe it is enough to include 
```#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"``` ,
possibly in the .cc rather than in the header

Reply:
ok, adding it in the .cc still gives me problems in the compilation. If I add this include in the .h instead of the EDProducer.h, the compilation is fine.

Reply:
Yes, it must stay here, sorry for the different suggestion in the previous note: ParameterSetDescription is needed here in the definition of the fillDescrption* methods.